dialog/information_test.go

```diff
@@ -51,7 +51,7 @@ func TestDialog_ErrorCallback(t *testing.T) {
 	func() {
 		select {
 		case <-tapped:
-		case <-time.After(1 * time.Second):
+		case <-time.After(100 * time.Second):
```

Comment:
Now you have raised the timeout to 100 seconds instead of lowering it to 100 milliseconds…

Reply:
Sorry again. Overlooked the milliseconds from the beginning. Not may day(s). Should now really be fixed.